main.groovy

```diff
@@ -0,0 +1,349 @@
+                if (use_wum.toBoolean()){
+                    aws_repo_branch="${product_version}-new"
+                    updateType="wum"
+                }else{
+                    aws_repo_branch="${product_version}-u2-new"
+                    updateType="u2"
+                }
+                dir("aws-"+product) {
+                    git branch: "${aws_repo_branch}",
+                    credentialsId: "JANETH_GITHUB_TEMP",
+                    url: "${cfn_repo_url}"
+                }
+            }
+        }
+    }
+    stage('Constructing parameter files'){
+        steps {
+            script {
+                withCredentials([string(credentialsId: 'AWS_ACCESS_KEY_ID', variable: 'accessKey'),
+                string(credentialsId: 'AWS_SECRET_ACCESS_KEY', variable: 'secretAccessKey'),
+                string(credentialsId: 'WUM_USERNAME', variable: 'wumUserName'),
+                string(credentialsId: 'WUM_PASSWORD', variable: 'wumPassword'),
+                string(credentialsId: 'DEPLOYMENT_DB_PASSWORD', variable: 'dbPassword'),
+                string(credentialsId: 'AWS_ACCESS_KEY_ID', variable: 's3accessKey'),
+                string(credentialsId: 'AWS_SECRET_ACCESS_KEY', variable: 's3secretKey')])
+                {
+                    sh '''
+                        echo "Writting AWS-Access Key ID to parameter file"
+                        ./scripts/write-parameter-file.sh "AWSAccessKeyId" ${accessKey} "${WORKSPACE}/parameters/parameters.json"
+                        echo "Writting AWS-Secret Access Key to parameter file"
+                        ./scripts/write-parameter-file.sh "AWSAccessKeySecret" ${secretAccessKey} "${WORKSPACE}/parameters/parameters.json"
+                        echo "Writting WUM Password to parameter file"
+                        ./scripts/write-parameter-file.sh "WUMPassword" ${wumPassword} "${WORKSPACE}/parameters/parameters.json"
+                        echo "Writting WUM Username to parameter file"
+                        ./scripts/write-parameter-file.sh "WUMUsername" ${wumUserName} "${WORKSPACE}/parameters/parameters.json"
+                        echo "Writting DB password to parameter file"
+                        ./scripts/write-parameter-file.sh "DBPassword" ${dbPassword} "${WORKSPACE}/parameters/parameters.json"
+                        echo "Writting S3 access key id to parameter file"
+                        ./scripts/write-parameter-file.sh "S3AccessKeyID" ${s3accessKey} "${WORKSPACE}/parameters/parameters.json"
+                        echo "Writting S3 secret access key to parameter file"
+                        ./scripts/write-parameter-file.sh "S3SecretAccessKey" ${s3secretKey} "${WORKSPACE}/parameters/parameters.json"
+                    '''
+                }
+                withCredentials([usernamePassword(credentialsId: 'JANETH_GITHUB_TEMP', usernameVariable: 'githubUserName', passwordVariable: 'githubPassword')]) 
```

Comment:
Replace credential ID here as well

Reply:
Fixed by replacing to the new id `WSO2_GITHUB_TOKEN`